src/main/kotlin/org/ionproject/integration/model/internal/timetable/Timetable.kt

```diff
@@ -0,0 +1,9 @@
+package org.ionproject.integration.model.internal.timetable
+
+data class Timetable(
+    var school: String = "",
+    var programme: String = "",
+    var calendarTerm: String = "",
+    var classSection: String = "",
+    var courses: List<Course> = listOf()
```

Comment:
Same as course

Reply:
They can't because they aren't filled all at the same time